packages/template-typescript-minimal/app/ssr.js

```diff
@@ -15,9 +15,8 @@ const options = {
     // The cache time for SSR'd pages (defaults to 600 seconds)
     defaultCacheTimeSeconds: 600,
 
-    // The path to the favicon. This must also appear in
-    // the mobify.ssrShared section of package.json.
-    faviconPath: path.resolve(__dirname, 'static', 'favicon.ico'),
+    // The location of the apps manifest file relative to the build directory
+    manifestPath: 'static/manifest.json',
```

Comment:
Is `manifestPath` being used at all? Looks at those it's not being used, I'm going to follow up and see if it's being used by the managed runtime unless you know more about this

Reply:
Not sure about `manifestPath`

Reply:
Kevin has a PR removed manisfestPath for retail template because it is not being used. We might want to do the same for other templates

Reply:
So as it turns out that [this](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/96c3f2c84564b54b30e232d43ec108ef53d2d346/packages/progressive-web-sdk/src/ssr/ssr-server.js#L804) is the last time we actually used that configuration value, way back in v1 (mobify version 1, like 3 versions ago) ... so you can get rid of all references of this option. And things looks good after that.